intezer_sdk/results.py

```diff
@@ -0,0 +1,104 @@
+from typing import List, Dict, Tuple
+
+from intezer_sdk.api import IntezerApi, raise_for_status
+
+
+class Results:
+    def __init__(self, url: str, api: IntezerApi, data: Dict):
+        """
+        Store list of data into pages.
+        :param url: Url to request new data from.
+        :param api: Istance of publice intezer api for request server.
+        :param data: filtered data.
+        """
+        self.api = api
+        self.data = data
+        self.pages = []
+        self._current_page = None
+        self._url = url
+        self._page_size = data["limit"]
```

Comment:
toy either use the pagination parameters from data or removes them completely, but not mix